crates/env/src/engine/experimental_off_chain/impls.rs

```diff
@@ -225,8 +225,8 @@ impl EnvBackend for EnvInstance {
         )
     }
 
-    fn println(&mut self, content: &str) {
-        self.engine.println(content)
+    fn debug_message(&mut self, message: &str) {
+        self.engine.debug_message(message)
```

Comment:
With the new print macros we might even think about getting rid of this since this is not needed and less powerful.

Reply:
Print macros depend on this  `Backend` method via https://github.com/paritytech/ink/blob/2e6ba6c60a1328996c953d3694a2698c4b26dc75/crates/env/src/api.rs#L515

Reply:
Ah sorry, I thought this piece of code was actually part of the `ink_lang::EnvAccess` getters that provide syntax for `self.env().println("message")`.